app/views/account/pages/_pages_list_row.html.slim

```diff
@@ -1,6 +1,10 @@
-tr
-  td
-    h3 = link_to page.title, controller: 'account/pages', action: 'show', id: page.id
-    = link_to page.source_url, page.source_url
-  td = page.status
-  td = page.rating
\ No newline at end of file
+      tr[id="#{dom_id(page)}"]
```

Comment:
it's better to add prefix for table rows, as `tr[id="#{dom_id(page, :table_row)}"]`